src/pages/Login/components/Login/LoginComponent.tsx

```diff
@@ -41,41 +45,49 @@ const LoginComponent = () => {
         navigate("/", { replace: true })
       })
       .catch(() => {
-        alert("잘못된 이메일이거나 잘못된 비밀번호의 조합입니다.")
+        setAlertMessage("잘못된 이메일이거나 잘못된 비밀번호의 조합입니다.")
+        showModal()
       })
   }
 
   return (
-    <S.LoginComponentLayout>
-      <S.LoginComponentTitle>로그인</S.LoginComponentTitle>
-      <S.LoginForm onSubmit={handleLogin}>
-        <LoginInputContainer
-          updateUserInfo={updateUserInfo}
-          type="email"
-          placeholder="이메일"
-        />
-        <LoginInputContainer
-          updateUserInfo={updateUserInfo}
-          type="password"
-          placeholder="비밀번호"
-        />
-        <S.ButtonContainer>
-          <S.Button
-            type="submit"
-            $color={theme.colors.point}
-          >
-            로그인
-          </S.Button>
-          <S.Button
-            type="button"
-            $color={theme.colors.sub_alt}
-            onClick={() => navigate("/signup")}
-          >
-            회원가입
-          </S.Button>
-        </S.ButtonContainer>
-      </S.LoginForm>
-    </S.LoginComponentLayout>
+    <>
+      <ModalAlert
+        isShow={isShowModal}
+        alertMessage={alertMessage}
```

Comment:
[P2] 요 부분은 제가 보기에는 "잘못된 이메일이거나 잘못된 비밀번호의 조합입니다." 라는 하나의 멘트로만 alert을 보여줄 것 같은데요! 그럼 state를 사용하고 직접 아래와 같이 값을 넣어주시면 불필요한 렌더링을 줄일 수 있겠네효

```suggestion
        alertMessage={"잘못된 이메일이거나 잘못된 비밀번호의 조합입니다."}
```

Reply:
좋은 의견 감사합니다!! 😊